README.md

```diff
@@ -108,10 +108,10 @@ We test different master/node image versions against multiple k8s branches on di
 
 Those jobs are using channel based versions, current supported testing map is:
 - k8s-dev : master
-- k8s-beta : release-1.11
-- k8s-stable1 : release-1.10
-- k8s-stable2 : release-1.9
-- k8s-stable3 : release-1.8
+- k8s-beta : release-1.12
```

Comment:
does the branch even exist yet? it doesn't look like it https://github.com/kubernetes/kubernetes/branches/all

Reply:
it will be cut this week

Reply:
it exists now